Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM iojs:latest
+
+
+#
+# installation
+#
+
+RUN mkdir /app
+RUN npm install -g grunt-cli coffee-script http-server
+
+ADD ./package.json /app/package.json
+RUN cd /app && npm install -q
```

Comment:
> copying the package.json first avoids time-consuming npm installs upon changes to the source-code. It will only install if package.json is affected.

I believe you're right. Let's leave it as is then.